app/Repositories/PhotoRepository.php

```diff
@@ -39,18 +40,22 @@ public function __construct(AWS $aws, Registrar $registrar)
      * @param  int $signupId
      * @return \Rogue\Models\Photo|null
      */
-    public function create(array $data, $signupId)
+    public function create(array $data, Signup $signup)
```

Comment:
Why do we need the whole `$signup` here? I believe that `northstar_id` should already be in the data array and all you really need is the `signupId` to associate with the item